runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -61,6 +77,12 @@
 	@Inject
 	private ItemManager itemManager;
 
+	@Inject
+	private KingdomConfig config;
+
+	@Inject
+	private ChatMessageManager chatMessageManager;
+
 	@Getter
 	private int favor = 0, coffer = 0;
```

Comment:
These can be removed in favor of using the config

Reply:
True. Changed in latest commit,